Files/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -709,8 +708,8 @@ public async Task<ReturnResult> MoveItemsFromClipboard(DataPackageView packageVi
 
         public async Task<ReturnResult> RenameAsync(IStorageItem source, string newName, NameCollisionOption collision, bool registerHistory)
         {
-            FileSystemStatusCode returnCode = FileSystemStatusCode.InProgress;
-            Progress<FileSystemStatusCode> errorCode = new Progress<FileSystemStatusCode>();
+            var returnCode = FileSystemStatusCode.InProgress;
+            var errorCode = new Progress<FileSystemStatusCode>();
```

Comment:
what's the purpose of changing to `var`s here?

Reply:
Not sure why that happened I don't remember even touching that method. Either way, I've pushed an update to fix this.

Reply:
@jusso-dev it's because of this commit:
https://github.com/files-community/Files/pull/3393/commits/2bf0c4e8187c14f94ea8c8ac3e5da6faaa43c704#diff-a278c320aed938dba6d11cd0d383d22875a893101a7b6c522c0054540d977dacR711

Reply:
@d2dyno1 No need to duplicate object type, it is better to use `var` here.

Reply:
Using `var`s for defining variables can be unclear. I prefer only using `var` in `foreach` loops

Reply:
Yes, it can be unclear, but this is not the case.

Reply:
So are you going to revert my change @tsvietOK ?

Reply:
This is my change:)

Reply:
Yeah but I reverted it so just check it looks the way you want (see my last commit)